assets/js/app.js

```diff
@@ -171,13 +171,6 @@ global.MonsieurBizRichEditorManager = class {
     let inputValue = this.input.value.trim();
 
     this.tags = this.input.dataset.tags.length === 0 ? [] : this.input.dataset.tags.split(',');
-    this.tagsAreExclusive = false;
-    for (let tag of this.tags) {
-      if (!tag.startsWith('-')) {
-        this.tagsAreExclusive = true;
-        break;
-      }
-    }
```

Comment:
This is very important. You need to understand the exclusivity.

`-tag` is different than `foo -tag`.

`-tag` alone means `all components without the `tag` tagged ones`.
`foo -tag` means `all foo tagged components without the `tag` tagged ones`.

And not `all elements, including foo ones, without `tag` tagged ones`.